src/plugins/data_source_management/server/plugin.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import {
+  PluginInitializerContext,
+  CoreSetup,
+  CoreStart,
+  Plugin,
+  Logger,
+} from '../../../core/server';
+
+import { dataSource } from './saved_objects';
+
+import { DataSourceManagementPluginSetup, DataSourceManagementPluginStart } from './types';
+
+export class DataSourceManagementPlugin
+  implements Plugin<DataSourceManagementPluginSetup, DataSourceManagementPluginStart> {
+  private readonly logger: Logger;
+
+  constructor(initializerContext: PluginInitializerContext) {
+    this.logger = initializerContext.logger.get();
```

Comment:
Since we imported logger, can we add some logs for each stage?

Reply:
A common one, is on start log the plugin has started.

Reply:
this part has been removed